IngestModules/Chrome Passwords Identifier/ChromePasswords.py

```diff
@@ -0,0 +1,247 @@
+# This module extracts the usernames, websites for Chrome and organizes it in the Tree Viewer Window

```

Comment:
The .py files should be in another folder. Something like:
IngestModules/Chrome Passwords Identifier/Chrome Password Identifier/ChromePasswords.py